client/src/research/Interactions.js

```diff
@@ -11,17 +12,37 @@ class Interactions extends Component {
   }
 
   render() {
+    const {email, token} = this.props;
+    const opt = {
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json',
+        'email': email,
```

Comment:
My intuition is that these header names should be formatted like `X-SwipeRight-Email` and the link.  I think this is based on other custom headers I've worked with in the past.  quick Googling shows some other discussion on this if you are curious too: https://stackoverflow.com/questions/3561381/custom-http-headers-naming-conventions

Reply:
hmmmmmm. so the gist that I am getting is to use X-SwipeRight before my custom headers to indicate they should not be used outside of this app and to make it harder for collisions with other headers that may already be used. Sound about right? I changed headers to `X-SwipeRight-Token` and `X-SwipeRight-Email` and updated the onlyAllowResearchers method

Reply:
note: headers are all lowercase.

Reply:
@keving17 Awesome!  What's the thinking on the header names being all lowercase?  It probably doesn't matter, just different than what I'm used to seeing.

Reply:
oh. So I originally had the headers uppercase like `X-SwipeRight-Token` and `X-SwipeRight-Email` but I noticed that even thought they were uppercase in the request in Interactions.js, when I looked at the request in browser they were all lowercase. I ended up running into a problem where the server could never find the token or email because it was looking for headers like `X-SwipeRight-Token` and `X-SwipeRight-Email` but the client was making requests with headers like `x-swiperight-token` and `x-swiperight-email`